src/SprykerMiddleware/Zed/Process/Communication/Console/ProcessConsole.php

```diff
@@ -29,13 +29,23 @@ class ProcessConsole extends Console
     protected const OPTION_ITERATOR_LIMIT = 'limit';
     protected const OPTION_LOG_LEVEL = 'flagLogLevel';
     protected const OPTION_INPUT = 'input';
+    protected const OPTION_INPUT_OPTIONS = 'input-options';
     protected const OPTION_OUTPUT = 'output';
+    protected const OPTION_OUTPUT_OPTIONS = 'output-options';
```

Comment:
Yes, it can be. I just wanted to make it a little bit shorter because of final command finally can be too long eg: `console middleware:process:run -p PROCESS_NAME --output-options '{"rootNodeName": "Root1", "entityNodeName": "Entity1"}' -o ./output.xml --input-options '{"rootNodeName": "Root"}' -i ./input.xml` and here are only required parameters for output and used shortcuts.

Reply:
IMO the length of an argument shouldn't matter. We have to be explicit and prevent others from mental mappings.